### DISCUSSION TITLE
How to plot a function?

### DISCUSSION THREAD

**Discussion by kevinmatthes** [PaddiM8/kalk#134] [Q&A] [answered]
First of all, thank you for this wonderful software!

I would like to become familiar with Kalker and therefore, I tried to compare its functionalities against GNU Octave / MATLAB.  One major feature I did not encounter, yet, was a plotting functionality.  I tried to search the documentation but I did not find something, yet.  What have I overlooked?

**Comment by PaddiM8** [comment, marked as answer]
Plotting functionality would be cool to have, but unfortunately it hasn't been implemented yet. Adding it to the web version would probably make sense. Maybe one day.

**Reply by kevinmatthes** [reply]
Thank you for your answer!

Just as a suggestion:  maybe the CLI could emit some Asymptote code or something similar as plot output?  If multiple formats would be supported, the user could decide for their preferred format for further processing.

**Reply by PaddiM8** [reply]
Ohh that would be neat
